hraven-etl/src/main/java/com/twitter/hraven/mapreduce/GraphiteHistoryWriter.java

```diff
@@ -372,6 +376,62 @@ public void write() throws IOException {
     socketWriter.write(lines.toString());
   }
 
+    private int getTimeStamp() {
+        int timestamp;
+
+//        timestamp = Math.round(recordCollection.getSubmitTime() / 1000);
+//        return timestamp;
+
+
+        StandardEvaluationContext context = new StandardEvaluationContext();
+
+        try {
+            context.registerFunction("dateToEpoch", GraphiteHistoryWriter.class.getDeclaredMethod("dateToEpoch", String.class));
```

Comment:
abstract out context generation for both getTimestamp and getMetricsPath into a separate method - getExpressionEvalContext. have a single context generated with all the methods and tokens. no need to repeat code.